conda/petsc/arm64_framework.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/config/BuildSystem/config/framework.py b/config/BuildSystem/config/framework.py
+index 214b1abef1..d63871e91f 100644
+--- a/config/BuildSystem/config/framework.py
++++ b/config/BuildSystem/config/framework.py
+@@ -550,6 +550,7 @@ class Framework(config.base.Configure, script.LanguageProcessor):
+       lines = [s for s in lines if s.find(' was built for newer macOS version') < 0]
+       lines = [s for s in lines if s.find(' was built for newer OSX version') < 0]
+       lines = [s for s in lines if s.find(' stack subq instruction is too different from dwarf stack size') < 0]
++      lines = [s for s in lines if s.find(' could not create compact unwind for') < 0]
```

Comment:
I will check if the new release has this fix. We do not want to do this kind of thing too often . It will be hard to maintain multiple versions of petsc.

Reply:
@fdkong did you look into a newer release? @milljm Do you have any comment on this ARM64 PETSc patch to address Fande's concern?

Reply:
@fdkong I would like to work on getting this merged into the framework today or tomorrow. Since ARM64 on MacOS is experimental for the moment, we will have a few more PRs coming to address M1 compatibility and hopefully your concerns about this patch. Do you have any other review comments to share in this PR?

Reply:
We can go this patch. I will patch future petsc

Reply:
Keep in mind that this is not in production. A package for arm64 is not currently being built. We might need to make changes (or remove this patch altogether) as we develop more for this platform.

Reply:
Very much a work in progress